src/pkg/backup/details/details.go

```diff
@@ -102,6 +102,21 @@ func (dm DetailsModel) Items() []*DetailsEntry {
 	return res
 }
 
+// Folders returns a slice of *ItemInfo that contains only FolderInfo entries.
+func (dm DetailsModel) Folders() []*DetailsEntry {
```

Comment:
Can I pester you for the inverse func as well?  (only non-folders).  I've already written some local filters to do that elsewhere.

Reply:
I believe `Items()` (above) is what you want?

Reply:
Oh.  Huh.  So it is.